build-script-helper.py

```diff
@@ -87,8 +106,17 @@ def get_swiftpm_options(
         args += ["--multiroot-data-file", multiroot_data_file]
     if verbose:
         args += ["--verbose"]
-    if platform.system() == "Darwin":
-        args += ["-Xlinker", "-rpath", "-Xlinker", "/usr/lib/swift"]
+    build_target = get_build_target(
+        swift_exec, cross_compile_config=cross_compile_config
+    )
+    build_os = build_target.split("-")[2]
+    if build_os.startswith("macosx"):
+        args += [
+            "-Xlinker",
+            "-rpath",
+            "-Xlinker",
+            "/usr/lib/swift",
+        ]
```

Comment:
Do we actually need either of the paths after this?

Reply:
I’m not sure. I wanted to start with the same paths that sourcekit-lsp uses and then try to simplify the paths again.